src/components/Book.js

```diff
@@ -1,21 +1,27 @@
-/* eslint-disable react/prop-types */
 import React from 'react';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 
 const Book = ({ bookList }) => (
```

Comment:
- [ ] This component should only display a single book data, and your `Books` component should display a list of `Book` components, kindly make this change 😄